data_diff/database.py

```diff
@@ -362,6 +435,10 @@ class Presto(Database):
         "timestamp": Timestamp,
         # "datetime": Datetime,
     }
+    NUMERIC_TYPES = {
+        "integer": Integer,
+        "real": Float,
```

Comment:
I think you're missing [double-precision](https://prestodb.io/docs/current/language/types.html#floating-point) here